delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/remote/client/RemoteDiskStorageClient.scala

```diff
@@ -32,7 +32,11 @@ import scala.concurrent.duration._
 /**
   * The client to communicate with the remote storage service
   */
-final class RemoteDiskStorageClient(baseUri: BaseUri)(implicit client: HttpClient, as: ActorSystem) {
+final class RemoteDiskStorageClient(baseUri: BaseUri)(implicit
```

Comment:
Now that the baseUri is the only thing that is not coming from the configuration, it feels like we could instantiate the remote client only once so as to pass it directly instead of its dependencies and building it each time.

The `baseUri` could become an implicit parameter for the different method